usr/usr.c

```diff
@@ -0,0 +1,73 @@
+#include <stdio.h>
+#include <assert.h>
+#include <windows.h> 
```

Comment:
We will have a Linux subsystem for Windows. :)

Reply:
> We will have a Linux subsystem for Windows. :)

Wow! LSW, that's amazine

Reply:
🤣This guy even hasn't asked AI what is linux.